### ISSUE TITLE
Change email to code

### ISSUE THREAD

**Issue by mcurmei627** [Algorithmic-Personalization/ytdpnl-server#1]
Here the console search function says e-mail instead of participant id

![Image](https://github.com/Algorithmic-Personalization/ytdpnl-server/assets/19541391/821c7fcc-404d-4355-8fe7-fe907535f32d)

**Comment by djfm** [comment]
indeed, leftover from when we had emails, it does search in codes though, will fix the label
